website/content/en/docs/contribution-guidelines/developer-guide.md

```diff
@@ -113,6 +113,36 @@ COPY watches.yaml ${HOME}/watches.yaml
 COPY helm-charts/ ${HOME}/helm-charts/
 ```  
 
+## How the operator-sdk images are built?
+
+In the Deploy stage configured in the [.travis.yml ][travis], the images are built and pushed to the `quay.io/operator-framework` repository. This stage is just executed against the master branch when a Pull Request is merged. 
+
+See that the images are built with different architectures from master as it is also done in the release process. See in the release Tab that each version has different architectures as well for each supported operator type such as Helm and Ansible for example. 
+
+In the release process, the script [.release.sh][release-sh] is executed and it will call the [makefile][makefile] target [make release](https://github.com/operator-framework/operator-sdk/blob/master/Makefile#L113) which will call the same targets which are used by the Travis in the master CI job to build these images. To know more about the release process, see the [doc][release-doc].
+
+## How to test the built of operator-sdk images? 
+
+If you are looking for to contribute to it, then you will need to ensure that the changes made in your Pull Request will also be executed successfully in the `Deploy` stage which, as described above, will just be trigged from the master branch.
```

Comment:
If you are looking to contribute to "it".

What types of changes need this extra step? Anything that changes the base image? Or anything that changes the automation?

Reply:
Any change that requires test the build of an operator-sdk image. E.g new type, changes in the process and etc .

Reply:
```suggestion
If your contribution alters a base image or build process, follow these steps to execute the Travis `Deploy` stage against your branch to demonstrate that the merge build will complete as expected.
```